docs/sources/setup/provisioning.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: '/provisioning'
+title: 'Provisioning'
+menuTitle: Provisioning
+description: Provisioning the GitHub data source plugin
+aliases:
+  - github
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+weight: 104
+---
+
+# Provisioning
+
+You can define and configure the GitHub data source in YAML files as part of Grafana’s provisioning system. For more information about provisioning a data source, and for available configuration options, see [Provision Grafana](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources).
```

Comment:
We shouldn't use product possessives like "Grafana's": https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanaproductpossessives

```suggestion
You can define and configure the GitHub data source in YAML files with Grafana provisioning.
For more information about provisioning a data source, and for available configuration options, refer to [Provision Grafana](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources).
```